oauth2client/contrib/sql_alchemy.py

```diff
@@ -0,0 +1,86 @@
+"""OAuth 2.0 utilities for SQLAlchemy.
+Utilities for using OAuth 2.0 in conjunction with SQLAlchemy.
+Heavily inspired by equivalent Django module.
+"""
+
+from oauth2client.client import Storage as BaseStorage
+from sqlalchemy.types import PickleType
+
+
+__author__ = 'dominik@mdzn.pl (Dominik Miedzinski)'
```

Comment:
I agree it does, although pretty much all modules contain it. We can solve this easily in another pull request, but at the moment I kind of insist on it.